src/nouislider.js

```diff
@@ -1100,6 +1111,53 @@
             return addNodeTo(handle.firstChild, options.cssClasses.tooltip);
         }
 
+        // create or update joint tooltips and hide or display related tooltips
+        function updateJointTooltips() {
+            for (var index in scope_OverlappingTooltips) {
+                var jointTooltip = scope_JointTooltips[index];
+                var overlapInfo = scope_OverlappingTooltips[index];
+
+                var firstHandleIndex = overlapInfo[0];
+                var secondHandleIndex = overlapInfo[1];
+
+                var firstTooltip = scope_Tooltips[firstHandleIndex];
+                var secondTooltip = scope_Tooltips[secondHandleIndex];
+
+                // if overlapping tooltips info has 3 values, two overlapping tooltips are separated and joint tooltip
+                // can be removed and original tooltips can be visible again
+                //
+                // else if overlapping tooltip is not existing then create it and gide related tooltips
+                //
+                // else update position of overlapping tooltip
+                if (overlapInfo.length === 3) {
+                    firstTooltip.className = options.cssClasses.tooltip;
+                    secondTooltip.className = options.cssClasses.tooltip;
+                    jointTooltip.remove();
```

Comment:
This remove method isn't supported in Internet Explorer:

https://caniuse.com/#feat=childnode-remove